theseus/optimizer/linear/__init__.py

```diff
@@ -3,7 +3,12 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import torch
+
 from .dense_solver import CholeskyDenseSolver, DenseSolver, LUDenseSolver
 from .linear_optimizer import LinearOptimizer
 from .linear_solver import LinearSolver
 from .sparse_solver import CholmodSparseSolver
+
+if torch.cuda.is_available():
```

Comment:
Flagging here too so we remember to make it consistent with what we end up doing in other places.